roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/publish/RoqFrontMatterPublishProcessor.java

```diff
@@ -116,7 +116,7 @@ public void paginatePublish(RoqSiteConfig config,
                     nextUrl = nextPage.url();
                 }
                 Paginator paginator = new Paginator(paginate.collection(), total, paginate.size(), countPages, i, prev,
-                        previousUrl, next, nextUrl);
+                        next, paginatedPages.get(0).url, nextUrl, previousUrl);
```

Comment:
```suggestion
                        next, nextUrl, previousUrl);
```

Reply:
use this as order:
int total,
int currentIndex,
RoqUrl firstUrl,
Integer previousIndex,
RoqUrl previous,
Integer nextIndex,
RoqUrl next